lib/rexml/parsers/baseparser.rb

```diff
@@ -505,15 +505,13 @@ def pull_event
       private :pull_event
 
       def entity( reference, entities )
-        value = nil
         value = entities[ reference ] if entities
         if value
           record_entity_expansion
-        else
-          value = DEFAULT_ENTITIES[ reference ]
-          value = value[2] if value
+          return unnormalize( value, entities ) if value
```

Comment:
Do we need this `if value`?

Reply:
No, you are right, it's unnecessary. Removed it in [b5da207](https://github.com/ruby/rexml/pull/199/commits/b5da2078bc7e7c6331ed3f7d8369a052d172c5fe)